message.go

```diff
@@ -27,9 +28,13 @@ type Message struct {
 	// ClientID identifies a particular session via a session id token
 	// See also https://docs.cometd.org/current/reference/#_bayeux_clientid
 	ClientID string `json:"clientId,omitempty"`
-	// Data contains the binary represnetation of the data
-	// See also https://docs.cometd.org/current/reference/#_data
-	Data *MessageData `json:"data,omitempty"`
+	// Data contains the event information and optionally could contain
+	// a binary representation of the data
+	// See also https://docs.cometd.org/current/reference/#_data and
+	// https://docs.cometd.org/current/reference/#_concepts_binary_data
+	//
+	// There is the MessageData type can be used for binary data
+	Data json.RawMessage `json:"data,omitempty"`
```

Comment:
I think I misread the docs in this particular case. I see two examples one where `data` at this level is just a string, and another where it has a nested object that has its own `data` field. I'm comfortable getting rid of `MessageData` altogether

Reply:
@sigmavirus24 so just json.RawMessage is okay and MessageData could be entirely removed?

Reply:
Yes, but I can do that in a follow-up here